doc/api_queries.rst

```diff
@@ -110,6 +114,11 @@
 
     Evaluate the query. Returns a list of matching :class:`dnf.package.Package` instances.
 
+  .. method:: unneeded(sack, yumdb, debug_solver=False)
```

Comment:
Ok, unneeded was used by repoquery, but in future it will be part of dnf. Therefore it can be marked as private function, but only after import of repoquery will be made. Also the question is how plugins-extras handles leaves? Or if they can use same functions like autoremove or repoquery.